stubs/networkx/networkx/utils/__init__.pyi

```diff
@@ -23,3 +24,5 @@ from networkx.utils.misc import (
 from networkx.utils.random_sequence import *
 from networkx.utils.rcm import *
 from networkx.utils.union_find import *
+
+config: NetworkXConfig  # Set by networkx/__init__.py
```

Comment:
This is the only "special thing" I had to do throughout this PR.

`networkx/__init__.py` contains the lines:
```py
config = utils.backends._set_configs_from_environment()
utils.config = utils.configs.config = config 
```